tests/core/test_grid.py

```diff
@@ -27,14 +26,6 @@ def test_volume(hexbeam):
     assert hexbeam.volume > 0.0
 
 
-@pytest.mark.skipif(not system_supports_plotting(), reason="Requires system to support plotting")
-def test_struct_example():
-    # create and plot structured grid
-    grid = examples.load_structured()
-    grid.plot(off_screen=True)  # basic plot
-    grid.plot_curvature(off_screen=True)
```

Comment:
According to codecov there are 2 lines being uncovered due to the PR, `PointGrid.plot_curvature()` due to this. Consider restoring this as well in the plotting tests.

Reply:
Shoot, I really thought I re-added this test under `tests/plotting/`... thanks for catching

Reply:
There's _two_ implementations of `plot_curvatire()`.

Reply:
Aha! Thank you 🤦🏻‍♂️